lib/assets/javascript/components/date_time_picker.js

```diff
@@ -1,3 +1,11 @@
+import $ from "jquery";
+import moment from "moment";
+import datetimepicker from "bootstrap-datetimepicker-npm/src/js/bootstrap-datetimepicker.js";
+
+window.moment = moment;
+(datetimepicker)($);
+window.$ = $;
```

Comment:
@pablobm This line is necessary for the tests to run because we use things like:

```
# spec/features/orders_form_spec.rb
page.execute_script(<<-JS)
        var date = moment("#{time_string}", "YYYY-MM-DD hh:mm:ss");
        $('[data-type="datetime"]').data("DateTimePicker").date(date);
      JS
```

I don't know how to import jQuery within a `execute_script`.
Maybe I could have capybara to click on the calendar instead of injecting JS.
Or I could use a very custom namespace such as `window.administrate_jQuery = $;`